changelog.d/6955.bugfix

```diff
@@ -0,0 +1 @@
+Update algorithm for computing the height of messages.
```

Comment:
this file should be called pr-6955.bugfix since it is not referencing an actual issue

Reply:
Yeah, my bad: https://github.com/vector-im/element-ios/pull/6955/commits/006e01c38d3de706741998ca91ff092fc99e4806